### DISCUSSION TITLE
Can we remove parameter `isOriginalFilename` from files API

### DISCUSSION THREAD

**Discussion by tenthe** [apache/streampipes#2528] [Dev]
## Observation
The parameter `isOriginalFilename`  doesn't appear to be utilized in the `FilesService` within the UI.

## Question
Uncertainty regarding the necessity of the parameter in `org.apache.streampipes.rest.impl.PipelineElementFile.getFile` after the changes in #2300.

## Suggestion
Consider assessing if `isOriginalFilename` is still required and if the methods `FileManager.getFileByOriginalName` & `FileManager.getFile` can be streamlined accordingly.

**Comment by bossenti** [comment]
Hey @tenthe,

I'm in favor of getting rid of `isOriginalFilename` after #2430.
In terms of functionality, I cannot detect any usage of the attribute `isOriginalFilename`.
So I'm fine with removing it.

The only aspect we should consider is that the query parameter `isOriginalFilename` is still used in the `FileApi` class. I'm not sure if you want to remove it there as well, but just in case.

**Reply by tenthe** [reply]
The class `FileApi` is a very good hint. We should deprecate it first before we remove it to avoid breaking client code.
However, the parameter no longer has any functionality so we can still remove it in the `REST API`, right? 
We should only keep a method in the `FileApi` class with the parameter isOriginalName, which we deprecate and remove after the next release. 

If that works for you, I would turn the discussion into an issue.

**Reply by bossenti** [reply]
Sounds good to me!
